de-lib/src/main/java/org/iplantc/de/theme/base/client/apps/widgets/AppLaunchViewDisplayStrings.properties

```diff
@@ -1 +1,4 @@
 deprecatedAppMask = This application has been deprecated. If you need access to it, please contact support@cyverse.org.
+hpcAppWaitTimes = <p> This application runs on the XSEDE, the US national supercomputing network. Wait times for your analyses (jobs) to submit and run may be up to several days for large memory applications. Please do not resubmit your job! If your job has not been submitted after several days, contact support@cyverse.org.</p> <p>Analyses run on this system enter a queue that includes many non-CyVerse users. Access to this powerful system is made available to CyVerse users at no cost through a grant from the National Science Foundation.</p><a href='https://pods.iplantcollaborative.org/wiki/x/MYOdAQ' target='_blank'> Apply for your own XSEDE allocation to gain access to additional resources </a></p>
```

Comment:
`If your job has not been submitted after several days...`
I think the science team came up with this blurb, but I'm wondering if it might be confusing since the jobs stay in a "Submitted" state for a long time and users are waiting for the next state after Submitted.